projects/batfish-client/src/org/batfish/client/Client.java

```diff
@@ -673,12 +735,38 @@ private boolean exit() {
    private void generateDatamodel() {
       try {
          ObjectMapper mapper = new BatfishObjectMapper();
-
          JsonSchemaGenerator schemaGenNew = new JsonSchemaGenerator(mapper);
          JsonNode schemaNew = schemaGenNew
                .generateJsonSchema(Configuration.class);
          _logger.output(mapper.writeValueAsString(schemaNew));
 
+//         Reflections reflections = new Reflections("org.batfish.datamodel");
```

Comment:
No, not yet. I got the code going but there are a bunch of other issues. I'll come back to this at a later point in time.